### PR TITLE
Places tip text above other components

### LINKED ISSUE #54: Lock hover text should appear over the code snippet
screen shot 2017-02-28 at 10 31 49 am

### PR DESCRIPTION
**Pull request by dane-johnson**
closes #54 
screen shot 2017-02-28 at 4 09 50 pm

### CODE REVIEW

File: src/components/LockButton.jsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ const LockButton = ({onClick, readOnly}) => {
       label={`Switch to ${ readOnly ? 'edit' : 'read-only' } mode`}
       onTouchTap={onClick}
       tooltip={toolTipText}
+      style={{zIndex: 5}}
```
Comment (Hopding) [inline src/components/LockButton.jsx]: I mentioned this on one of Jordan's PRs as well: I think we should avoid inline styling in this project, and use CSS files/classes instead.

Comment (dane-johnson) [inline src/components/LockButton.jsx]: Might be against best-practice on this one.
http://www.material-ui.com/#/customization/styles

Comment (dane-johnson) [inline src/components/LockButton.jsx]: @Hopding In lieu of our discussion I invite you to reconsider requesting changes.

**Comment by Hopding** [comment]
:shipit:

**Pull request merged**